app/modules/ads/setup.js

```diff
@@ -147,6 +147,14 @@ function setupAdContext(adsContext, instantGlobals, isOptedIn = false) {
     context.set('custom.isCMPEnabled', true);
   }
 
+  const btlConfig = instantGlobals.wgAdDriverBillTheLizardConfig;
+
+  if (context.get('options.slotRepeater') && btlConfig && btlConfig.projects && btlConfig.projects.cheshirecat
+    && btlConfig.projects.cheshirecat.some(model => utils.isProperGeo(model.countries))) {
+    context.set('slots.incontent_boxad_1.insertBeforeSelector',
+      `${context.get('slots.incontent_boxad_1.insertBeforeSelector')},.article-body h3`);
```

Comment:
Doesn't it mean slots will be inserted before the selector set in the context or `.article-body h3`? I think this is what we want and we can't just change the setting in context but I want to double-check if I think correctly ;)

Reply:
We need `.article-content > h3`, otherwise it will jump into infobox ;)
https://github.com/Wikia/mobile-wiki/pull/1149/files

Reply:
@jacekbj Thanks, changed!
@nandy-andy That's exactly right and that's enough. Slot repeater and slot injector will use this selector to mark places where FMR X slot can be injected and adding H3 will increase the number of possible places.